src/tensors/gpu/backend.h

```diff
@@ -29,9 +29,19 @@ class Backend : public marian::Backend {
 
   cublasHandle_t getCublasHandle() { return cublasHandle_; }
 
+#if CUDA_VERSION >= 9000
+  bool useTensorCores() {
+    return useTensorCores_;
+  }
```

Comment:
Wouldn't it be better to have this function always defined and just returning `false` if CUDA < 9.0? That would possibly minimize the uses of `#if` statements in other places. 

If CUDA_VERSION implies CUDA_FOUND, maybe put this here explicitly?

Reply:
yes, that's probably better, I am going to change this.